packages/vue/src/examples/pages/home/Home.vue

```diff
@@ -331,7 +334,9 @@ export default {
   @include for-desktop {
     margin: 0;
   }
-  // todo: fix product shadow on carousel
+  &__item {
+    margin: var(--spacer-base) 0;
```

Comment:
```suggestion
    margin: 31px 0 39px 0; // convert px to rem 
```
because: you have 35px shadow with offset-y=4px